client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -0,0 +1,86 @@
+import React , {useCallback, useState} from 'react';
+import {Button, Card, InputGroup, FormControl} from 'react-bootstrap';
+import {setCalibration as sendCalibrationValue, resetCalibration} from 'api/common/powerModel';
+// import {useSensorData} from 'api/v2/sensors/data';
+
+/**
+ * Boost distance claibration input form
+ * 
+ * @returns {React.Component} Component
+ */
+function BoostCalibrationInput() {
+    const [calibrationValue, setCalib] = useState('');
+
+    const handleCalibrationChange = useCallback(
+        (event) => {
+        setCalib(event.target.value);
+        },
+        [setCalib],
+    );
+
+    const onSet = useCallback(
+        () => {
+          const calibValue: number = parseInt(calibrationValue, 10);
+          if (Number.isNaN(calibValue)) {
+            alert("Please enter an integer value only!");
+          }
+          else {
+            console.log(calibValue);
+            sendCalibrationValue(parseInt(calibrationValue, 10));
+          }
+          setCalib('');
+        },
+        [calibrationValue, setCalib],
+    );
+
+    const handleKeyPressed = useCallback(
+        (event) => { if (event.key === 'Enter') onSet(); },
+        [onSet]
+    );
+
+    return (
+    <InputGroup className="mb-1" >
+        <FormControl
+            onChange={handleCalibrationChange}
+            onKeyPress={handleKeyPressed}
+            placeholder="Calibrate distance..."
+            value={calibrationValue}
+        />
```

Comment:
Have a look at the form validation stuff here: https://react-bootstrap.github.io/components/forms/#forms-validation

Rather than an alert, it would be better to highlight the form red and say something like "Distance must be numerical". You could even do this validation live while the user is typing.

Also consider adding `type="number"` which tells it what you intend to use the form for.